dev-lang/retro/retro-2021.4.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="Clean, elegant, and pragmatic dialect of Forth"
+DESCRIPTION="Retro is a clean, elegant, and pragmatic dialect of Forth. \
+It provides a simple alternative for those willing to make a break from legacy \
+systems.
+
+The language draws influences from many sources including traditional \
+Forth systems, cmForth, colorForth, Factor, and Parable. It was \
+designed to be easy to grasp and adapt to specific uses.
+
+The basic language is very portable. It runs on a tiny virtual \
```

Comment:
I think this paragraph can be dropped.

Reply:
I agree, but this was copied verbatim from the original website (https://forthworks.com/retro/) and I've noticed that other OS's and distro packages put it without changes. Would you insist on cutting this paragraph still? @korli

Reply:
Personally I don't care, but others do: https://discuss.haiku-os.org/t/a-few-thoughts-on-recipes-summary-description/10598
"Remove features not available for Haiku. Don’t bore us with the history of the development or license changes etc. If it’s a game, describe the game play etc. but don’t dwell on who bought what and published it when and how…"

Reply:
fixed